axis_backend/.github/workflows/security-scan.yml

```
@@ -0,0 +1,162 @@
+name: Security Scan
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)